clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -5122,6 +5122,41 @@ def CIR_AtomicClearOp : CIR_Op<"atomic.clear"> {
   }];
 }
 
+def CIR_SyncScopeKind : CIR_I32EnumAttr<"SyncScopeKind", "sync scope kind", [
```

Comment:
Move this attribute closer to `CIR_AtomicFetchKind`

Reply:
done

Reply:
This hasn't been addressed.